src/nnfusion/engine/profiler/profiler.cpp

```diff
@@ -20,6 +21,23 @@ using namespace nnfusion::profiler;
 using namespace std::chrono;
 
 DECLARE_bool(fmerge_prof_compiling);
+DECLARE_bool(fantares_mode);
+
+static bool register_antares_kernel(std::string op_name)
+{
+    kernels::KernelRegistrar kernel_registrar_cuda(
+        op_name,
+        kernels::Name(op_name)
+            .Device(CUDA_GPU)
```

Comment:
Do we need to check the device？Maybe not CUDA

Reply:
Another way is following kernel_tuning pass, register antares kernels on CUDA+ROCM+CPU for any op.